src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 17px;
+  padding-top: 33px;
+  padding-bottom: 18px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 30px;
+  }
+
+  &__title {
+    justify-content: center;
+    font-family: $bodyFont;
+    color: $bodyColor;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+
+    width: 100%;
+    margin-bottom: 4px;
+  }
+
+
+  &:hover .card__title {
```

Comment:
The `.card__title` is already nested within `.card` and doesn't need to be specified again for the hover state. You can simply use `&:hover &__title` to achieve the same effect. This keeps the code DRY and easier to maintain.